.travis.yml

```
@@ -0,0 +1,22 @@
+dist: xenial
+sudo: false
+language: rust
+
+rust:
+- stable
+
+# https://levans.fr/rust_travis_cache.html
+# Need to cache the whole `.cargo` directory to keep .crates.toml for `cargo update` to work...
+# ...but don't cache the cargo registry
+cache:
+  directories:
+    - /home/travis/.cargo
+before_cache:
+  - rm -rf /home/travis/.cargo/registry
+
+env:
+  global:
+    - RUST_BACKTRACE="1"
+
+script:
+  cargo test --all --verbose
```

Perhaps we could also include the format and clippy :smile: 
```suggestion
  cargo fmt --all -- --check
  cargo clippy
  cargo test --all --verbose
```